en/components/dock-manager.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Manage Angular Page Layout | Dock Manager | Ignite UI | Infragistics
+_description: Learn how our Dock Manager can help you in customizing the page layout of your Angular application by using unpinning, resizing, moving and hiding panes.
```

Comment:
```suggestion
_description: The Dock Manager provides functionality for laying out your Angular application in panes that can be pinned, resized, moved, and hidden.
```

Reply:
The description is generated following our SEO process and should remain the same.

Reply:
I still think we can make it shorter than it is

Reply:
@StefanIvanov I think that the description you suggested is better than the one I initially composed

Reply:
@StefanIvanov Since the SEO guidelines we are following require using call to action words, I think about changing the beginning to 

> Learn how the Dock Manager [...]

Reply:
OK I am fine with that